overlay/usr/local/bin/jenkins

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+java -jar /usr/local/share/jenkins/jenkins-cli.jar $@
```

Comment:
I have no issue with this being an overlay, but there is a very similar (perhaps duplicate?) script (`/usr/local/bin/jenkins-cli`) created in `30cli`. I do notice that they have different names (`jenkins-cli` vs `jenkins`) but otherwise appear identical?! My inclination would be to just keep one and ditch the other.